provisioning/ubuntu-provision.sh

```diff
@@ -672,6 +672,22 @@ function cleanup() {
   sync
 }
 
+## Install rngd to increase VM entropy
+function install_rngd() {
+  apt-get update --quiet
+  apt-get install --yes --no-install-recommends rng-tools5
+  systemctl enable rng-tools5 || true
```

Comment:
```suggestion
  systemctl enable rng-tools5 || true # Uses logical operator || to prevent script from failing due to non-zero exit status in the case of docker containers
```